flutter/lib/ui/painting/image_decoding.cc

```diff
@@ -130,12 +129,12 @@ void DecodeImageFromList(Dart_NativeArguments args) {
   PassOwnPtr<DartPersistentValue> callback =
       adoptPtr(new DartPersistentValue(DartState::Current(), callback_handle));
 
-  RefPtr<SharedBuffer> buffer = SharedBuffer::create();
-  buffer->append(reinterpret_cast<const char*>(list.data()),
-                 list.num_elements());
+  const char* bytes = reinterpret_cast<const char*>(list.data());
+  PassOwnPtr<std::vector<char>> buffer =
```

Comment:
Generalized lambda capture would avoid the need for the pointer right? Are we switching to C++14? I thought I saw some GN files with that flag.